client/src/pages/UpdateGoal.tsx

```diff
@@ -202,7 +202,8 @@ function UpdateGoal() {
       }
     } else {
       // Not done - Update progress
-      await OWServiceProvider.updateGoal(tempGoalId, notes, newPagesRead);
+      // TODO: create "completed" state to manage whether goal has been completed and should be updated accordingly
```

Comment:
Just want to highlight this TODO. Right now "completed" variable is hardcoded as false in the updateGoal call